requirements.txt

```diff
@@ -1,4 +1,4 @@
 toml==0.10.2
-#git+https://github.com/nmdp-bioinformatics/py-graph-imputation
+py-graph-imputation==0.0.12
```

Comment:
could it be >=0.0.12 ?

Reply:
It's best to use the version it was tested against. It could be `>=0.0.12` if any/all future releases don't break the API.